susemanager-utils/testing/automation/update-obs-environment-cr-project.sh

```diff
@@ -32,11 +43,29 @@ if [ $# -eq 2 ];then
     test_parent_project=${2}
 fi
 
+parent_repo_name="openSUSE_Leap_15.3"
+
+new_project_config="
+    <project name=\"__TEST_PROJECT_NAME__\">
+  <title>Continuous Rebuild project for environment ${1}</title>
+  <description>This links to __PARENT_PROJECT_NAME RPMs. It will be disabled before each CI run, so there are no rebuilds during the CI build. Then, it will be enabled back to get the latest updates.</description>
+  <person userid=\"jordimassaguerpla\" role=\"maintainer\"/>
+  <person userid=\"juliogonzalezgil\" role=\"maintainer\"/>
```

Comment:
Ok, we'll need to change this later. We really need to start using a group.

Reply:
agree